lib/stm32/l0/rcc.c

```diff
@@ -290,5 +309,183 @@ void rcc_wait_for_osc_ready(enum rcc_osc osc)
 	}
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Get the System Clock source.
+ *
+ */
+enum rcc_osc rcc_get_sysclk(void) {
+	uint32_t	clk;
+	clk = (RCC_CFGR >> 2) & 0x3;
+	switch (clk) {
+	case 0:
+		return MSI;
+	case 1:
+		return (RCC_CR & RCC_CR_HSI16DIVF) ? HSI4 : HSI16;
+	case 2:
+		return HSE;
+	case 3:
+		return PLL;
+	}
+	return MSI;	/* not reached */
```

Comment:
Fixed by combining the case 0: case into a default: case which the compiler recognizes will never get past the switch statement.